pkg/blob/nodeserver.go

```diff
@@ -409,6 +409,10 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 	}
 
 	// Get mountOptions that the volume will be formatted and mounted with
+	if protocol == Fuse2 {
+		// Adding telemetry tag to know that blob is been mounted through AKS
+		mountFlags = util.JoinMountOptions(mountFlags, []string{fmt.Sprintf("--telemetry=azpartner-aks/%s", d.Version)})
```

Comment:
follow the PR example here: https://github.com/kubernetes-sigs/blob-csi-driver/pull/2024 since protocol may be empty here, and also check whether this option is already specified